<launch>

  <arg name="output" default="screen"/>
  <arg name="mocap_topic" default="/mocap_tracking"/>
  <arg name="mocap_frame_name" default="/mocap_world"/>
  <arg name="right_arm_pose_topic" default="/r_arm_pose_controller/pose" />

  <!-- Kill the /robot_pose_ekf node running on the robot -->
  <node pkg="mocap_robot_pose" type="kill_pr2_ekf.sh" name="kill_pr2_ekf" />

  <node pkg="tf2_ros" type="static_transform_publisher" name="odom_to_mocap_link"
      args="0 0 0 0 0 0 $(arg mocap_frame_name) /odom_combined" >
  </node>

  <node pkg="mocap_robot_pose" type="mocap_robot_base_pub" name="mocap_robot_base_pub" output="$(arg output)">
    <param name="mocap_topic" type="string" value="$(arg mocap_topic)" />
    <param name="robot_object_name" type="string" value="ArchieBase" />
    <param name="robot_segment_name" type="string" value="ArchieBase" />
  </node>

  <node pkg="mocap_robot_pose" type="mocap_robot_right_arm_pub.py" name="mocap_robot_right_arm_pub" output="$(arg output)">
    <param name="torso_frame" type="string" value="/torso_lift_link" />
    <param name="wrist_frame" type="string" value="/mocap_right_wrist" />
    <param name="right_arm_pose_topic" type="string" value="$(arg right_arm_pose_topic)" />
  </node>

</launch>
